spinq10q.yml

```diff
@@ -0,0 +1,274 @@
+nqubits: 10
```

Comment:
This information is definitely redundant, since you're listing them anyhow in the `qubits:` field. But maybe it should be 5.

Should we drop this key in 0.2? @stavros11

Reply:
> Should we drop this key in 0.2? @stavros11

Yes, we could drop. However looking at https://github.com/qiboteam/qibolab/blob/c566eace2ba4b6bc165c3411fb8635b31f20a7f9/src/qibolab/platform/platform.py#L126 I believe this already never loaded in the platform, so it would work even if we remove it (I did not test). If that’s the case, it’s just a matter of dropping from the runcards.